src/wallet/rpcwallet.cpp

```diff
@@ -583,6 +583,57 @@ static UniValue signmessage(const JSONRPCRequest& request)
     return EncodeBase64(vchSig.data(), vchSig.size());
 }
 
+static UniValue GetReceived(interfaces::Chain::Lock& locked_chain, CWallet * const wallet, const UniValue& params, bool by_label) EXCLUSIVE_LOCKS_REQUIRED(wallet->cs_wallet)
```

Comment:
style-nit: Instead of returning an UniValue, I'd prefer to return a `CAmount`, so that (if needed) the function can also be called from unit tests easily.

Reply:
Done

Reply:
@MarcoFalke can you expand on how returning `CAmount` makes it easier to test? Is it due to not having `UniValue` as a dependency? In that case I should remove `UniValue` as a parameter as well.